pandas/core/series.py

```diff
@@ -1429,8 +1429,55 @@ def mode(self):
         # TODO: Add option for bins like value_counts()
         return algorithms.mode(self)
 
-    @Appender(base._shared_docs['unique'] % _shared_doc_kwargs)
     def unique(self):
+        """
+        Return unique values of Series object.
+
+        Uniques are returned in order of appearance. Hash table-based unique,
+        therefore does NOT sort.
+
+        Returns
+        -------
+        unique values.
+          - If the input is an Index, the return is an Index
```

Comment:
This part can be simplified, since it is now only the docstring of Series.unique. 
So there is also no "input"

In this case it is always an `array`, except for categorical dtype.

Reply:
done.